filament/src/materials/skybox.mat

```diff
@@ -40,7 +40,7 @@ material {
     vertexDomain : device,
     depthWrite : false,
     shadingModel : unlit,
-    variantFilter : [ skinning, shadowReceiver ],
+    variantFilter : [ skinning, shadowReceiver, vsm ],
```

Comment:
Is this a safe change? I had to remove the VSM filter because the GltfViewer was crashing under certain conditions I have already forgotten (VSM + sunlight + shadowmap at 4K or something).

Reply:
This is probably a merge mistake. I had no crashes, but to be safe, I removed it in a9c4210